addons/info/README.md

```diff
@@ -130,7 +130,7 @@ setDefaults({
   inline: true, // Displays info inline vs click button to view
   source: true, // Displays the source of story Component
   propTables: [/* Components used in story */], // displays Prop Tables with this components
-  propTablesExclude: [], // Exclude Components from being shown in Prop Tables section
+  propTablesExclude: [], // Exclude Components from being shown in Prop Tables section. Pass the imported component names as values to the array.
```

Comment:
Component name is a string, but here the component itself (class or stateless function) should be passed